front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box } from '@citizenlab/cl2-component-library';
+import { colors } from 'utils/styleUtils';
+
+import { useNode, useEditor, ROOT_NODE } from '@craftjs/core';
+import { FormattedMessage } from 'utils/cl-intl';
+import { getComponentNameMessage } from '../../utils';
+
+export const RenderNode = ({ render }) => {
```

Comment:
The render node wraps each craft component in the preview but is not present in the serialized output. We use it to display the names of the selected components when a user clicks on them.